test/protocols/_waitfor.cxx

```diff
@@ -0,0 +1,74 @@
+/********************************************************************************
+ * Copyright (C) 2014-2018 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+
+#include "runner.h"
+
+#include <gtest/gtest.h>
+#include <boost/process.hpp>
+
+#include <sys/types.h>
+#include <signal.h>
```

Comment:
I guess those are not used?

Reply:
those are needed for the kill.

Reply:
Perhaps I am missing it, like with the `id()`, but I only see `terminate()` which sends SIGKILL. I want to have a SIGINT or SIGTERM to give the process a chance to handle it.